web/src/components/CountriesCell/CountriesCell.js

```diff
@@ -2,6 +2,19 @@ import { find, orderBy, filter, concat } from 'lodash'
 import Spinner from 'respin'
 import theme from 'src/theme'
 
+const variables = {}
+
+// https://redwoodjs.com/docs/cells#beforequery
+export const beforeQuery = () => {
+  return {
+    // never change query with new variables
+    variables,
```

Comment:
the query (below) never uses variable, so this prevents it from fetching what it thinks is a new query